.circleci/config.yml

```diff
@@ -403,7 +403,6 @@ jobs:
                 echo "Skipping - not running in INSTANCE_TESTS build"
                 exit 0
             fi
-            ./Tests/scripts/slack_notifier.sh $CIRCLE_NODE_INDEX ./env_results.json
```

Comment:
why did you remove this?

Reply:
I didn't want to spam the content team channel so I removed the notifications while I fixed this. Good thing you caught it.